.github/ISSUE_TEMPLATE/mojo_bug_report.yaml

```diff
@@ -1,6 +1,6 @@
-name: Beta Bug report
-description: Create a bug report for the beta program to help us improve Mojo
-title: "[BETA]: "
+name: Mojo bug report
```

Comment:
Should we be consistent with "bug report" and "Issue Report"? Not a big deal - but good to change if you can.

Reply:
I deliberately made them distinct as for Mojo we have both bug and enhancement templates.